cli/cmd/faucet.go

```diff
@@ -96,6 +96,9 @@ func faucet(cmd *cobra.Command, address []string) error {
 
 	var dat map[string]string
 	json.Unmarshal([]byte(data), &dat)
```

Comment:
you should handle the error returned by the json.Unmarshal

if err := json.Unmarshal([]byte(data), &dat); err != nil {
   // pretty print an error message
}